src/utilities/index.ts

```diff
@@ -0,0 +1,4 @@
+export {getValueForScreenSize} from './getValueForScreenSize';
+export {getResponsiveValue} from './getResponsiveValue';
+export {isResponsiveObjectValue} from './isResponsiveObjectValue';
+export {getThemeValue} from './getThemeValue';
```

Comment:
I split the `responsiveHelpers` file into multiple ones as it was getting too long